rpc/answerer_stats.go

```diff
@@ -1,93 +0,0 @@
-package rpc
-
-import (
-	"fmt"
-	"sync"
-	"time"
-
-	"github.com/viamrobotics/webrtc/v3"
-
-	"go.viam.com/utils"
-)
-
-// timeFormatStr copied from DefaultTimeFormatStr in RDK.
-const timeFormatStr = "2006-01-02T15:04:05.000Z0700"
-
-// answererStats is a collection of measurements/information gathered during
-// the course of a single connection establishment attempt from a signaling
-// answerer. We log the contents of the struct as a single INFO log message
-// upon successful or failed connection establishment to avoid emitting dozens
-// of logs during the answering process and cluttering regular robot logs.
-// Answerer stats are only logged in production for external signalers.
-type answererStats struct {
-	// mu guards all fields on answererStats.
-	mu sync.Mutex
-
-	success bool
-
-	// Stats below will be logged.
-	answerRequestInitReceived *time.Time
-	numAnswerUpdates          int
-	// "local" ICE candidates are those generated by the signaling answerer, so
-	// these represent server-side candidates.
-	localICECandidates []*localICECandidate
-	// "remote" ICE candidates are those received from the caller, so these
-	// represent client-side candidates.
-	remoteICECandidates []*remoteICECandidate
```

Comment:
`localICECandidates` and `remoteICECandidates` are now logged whenever the ICE connection state changes (for both callers and answerers). `numAnswerUpdates` and `answerRequestInitReceived` did not seem like useful stats to me.